wcomponents-core/src/main/java/com/github/bordertech/wcomponents/AbstractWComponent.java

```diff
@@ -1517,8 +1410,6 @@ public Container getParent() {
 
 	// ----------------------------
 	/**
-	 * {@inheritDoc}
```

Comment:
Please retain this inheritDoc tag as the inheritance needs to be explicit due to the other comments in the block.